packages/patternfly-4/react-styles/scripts/copyStyles.js

```diff
@@ -1,6 +1,7 @@
-/* eslint-disable no-case-declarations */
+/* eslint-disable @typescript-eslint/no-var-requires */
 const { copySync } = require('fs-extra');
 const { resolve, dirname, join } = require('path');
+/* eslint-enable @typescript-eslint/no-var-requires */
```

Comment:
This is more of a build script. Should we apply linter rules here? 

Perhaps we should create a different config for js?